ballot/models.py

```diff
@@ -52,6 +52,10 @@ def __str__(self):
 class BallotItemResponse(models.Model):
     """
     YES/NO to a referendum, or a candidate.
+
+    This is a *conceptually* abstract class, but exists
+    as a Django model so that finance can point to propositions
+    and candidates equally.
     """
```

Comment:
>So, just talking about referenda...why does the class have ... both the “pro” and the “con” statements?

The original reason was to enable multiple-choice referenda (which at least Portland has, and I'm not clear if other places do). But I've dropped that--only candidates have `pro_statement` and `con_statement` now.
>Really I think all the fields currently in this class aught to be somewhere else, maybe in the BallotItem itself or in a BallotItemDescription abstract class which can be subclassed to a BallotMeasureDescription and a BallotOfficeDescription.

That's basically what I just did, from your previous feedback... just using different class names. So, we're getting close! :)

>I think the name BallotItemChoice is better than BallotItemResponce as the ballot contains choices, not responses.

I had `BallotItemChoice` until I heard that the [Voter Information Project](https://votinginfoproject.org/) is using `BallotResponseItem`, and I wanted to be a bit more consistent. I agree that "choice" feels a bit more appropriate than "response"... I'm happy to change. @adborden any thoughts?